### DISCUSSION TITLE
Is there any way to open the IFC file if it is too large to open file for reading

### DISCUSSION THREAD

**Discussion by u0416033** [IfcOpenShell/IfcOpenShell#3170] [Q&A] [answered]
I have try to open the file that is over 300MB with ifcopenshell . It return  "OSError: Unable to open file for reading" . Is there any way to seperate the file and read the file? 
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/53708536/088d7cd6-cba7-4b91-a7d9-365491d41d64)

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/53708536/7577f42f-85b1-4722-9870-9b7b2cee1de1)

**Comment by Moult** [comment, marked as answer]
300MB is still pretty small and so I think you have a different issue. Can you check your filepaths, make sure if you're on windows to use / instead of \ for path separators, and of course check permissions. If you still have issues I'm happy to do a screenshare to help debug.

**Reply by u0416033** [reply]
@Moult Thank you ! I find that I have wrong file path when I check my code again.
